pihole/datadog_checks/pihole/pihole.py

```diff
@@ -0,0 +1,97 @@
+import requests
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+
+class PiholeCheck(AgentCheck):
+    @staticmethod
+    def _collect_response(url):
+        response = requests.get(url)
+        data = response.json()
+        status_code = response.status_code
+        return data, status_code
+
+    def check(self, instance):
+        host = instance.get('host')
```

Comment:
please do not use the instance param on the check method. It is deprecated. You can use `self.instance` instead